lib/states.js

```diff
@@ -60,7 +60,7 @@ export default class States {
       {state: method}
     ).then(results => {
       if (errors.length > 0) {
-        throw new Error(errors.map(e => e.toString()).join(' | '))
+        throw new Error(errors.map(e => (e.toString() + e.stack)).join(' | '));
```

Comment:
I'm not sure if the stack trace should be included here.  These errors get caught in task.js and end up in the log.  We should log these errors and stack traces to debug and remove things like this: https://github.com/taskcluster/docker-worker/blob/master/lib/task.js#L585